WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/SinglePackageHazmatDeclaration.swift

```diff
@@ -78,47 +79,69 @@ struct SinglePackageHazmatDeclaration: View {
                 .calloutStyle()
 
             Spacer()
-            Text(Localization.hazmatInstructionsSecondSection)
-                .calloutStyle()
+            createText(withLink: Localization.hazmatInstructionsSecondSectionLink,
+                       url: WooConstants.URLs.uspsInstructions.asURL(),
+                       content: Localization.hazmatInstructionsSecondSection)
 
             Spacer()
-            Text(Localization.hazmatInstructionsThirdSection)
-                .calloutStyle()
+            createText(withLink: Localization.hazmatInstructionsThirdSectionLink,
+                       url: WooConstants.URLs.uspsSearchTool.asURL(),
+                       content: Localization.hazmatInstructionsThirdSection)
 
             Spacer()
-            Text(Localization.hazmatInstructionsFourthSection)
-                .calloutStyle()
+            createText(withLink: Localization.hazmatInstructionsFourthSectionLink,
+                       url: WooConstants.URLs.dhlExpressInstructions.asURL(),
+                       content: Localization.hazmatInstructionsFourthSection)
 
             Spacer()
         }
         .padding(.leading, Constants.horizontalPadding)
         .padding(.trailing, Constants.longTextTrailingPadding)
     }
+
+    func createText(withLink linkText: String, url: URL, content: String) -> some View {
```

Comment:
super nit: this function can be private?

Reply:
Yes! 😄 Fixed in https://github.com/woocommerce/woocommerce-ios/pull/10598/commits/f4ebe9d5e382df8e6ff47a970e1a9e4ba17f34e0